### ISSUE TITLE
Performance issue with large spec

### ISSUE THREAD

**Issue by cmcmannus** [Redocly/redoc#1615]
I have implemented ReDoc using the standalone file served from the CDN (https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js) using a rather large spec (1.86MB), and once it finally renders, the resulting page is basically unusable.  Interactions on the page take >1s.  The page can be viewed [here](https://dirkelfman.github.io/other_samples/redoc/).

**Comment by adamaltman** [comment]
Yes, we only have a solution (pagination) for this issue in our [premium product](https://redoc.ly).
